### PR TITLE
fix: change in task end date and start date does not reflect in project

### PR DESCRIPTION
**Pull request by Anurag810**
fix: change in task end date and start date does not reflect in project

### CODE REVIEW

**Review by scmmishra — DISMISSED** [review]

File: erpnext/projects/doctype/task/task.py [hunk 1]
```diff
@@ -99,7 +99,14 @@ def update_time_and_costing(self):
 
 	def update_project(self):
 		if self.project and not self.flags.from_project:
-			frappe.get_doc("Project", self.project).update_project()
+			project = frappe.get_doc("Project", self.project)
+			for task in project.tasks:
+				if task.task_id == self.name:
+					task.start_date = self.exp_start_date
```
Comment (netchampfaris) [inline erpnext/projects/doctype/task/task.py]: This should be handled here

https://github.com/frappe/erpnext/blob/3866e132f71dfe98cb2dc95bf613a2b1d5de1cb7/erpnext/projects/doctype/project/project.py#L142

**Review by netchampfaris — CHANGES_REQUESTED** [review]

File: erpnext/projects/doctype/task/task.py (hunk 1 above)
Comment (Anurag810) [reply erpnext/projects/doctype/task/task.py]: but it synced when task is updated so put it into task.py

**Review by nabinhait — CHANGES_REQUESTED** [review]
Travis tests for Task are failing.

File: erpnext/projects/doctype/task/task.py (hunk 1 above)
Comment (Anurag810) [reply erpnext/projects/doctype/task/task.py]: it sync the tasks if something is updated from project side.
But Here is the opposite scenario when we update project the changes go to tasks.
But when we update the task(date) it does not go to the project.

Comment (Anurag810) [reply erpnext/projects/doctype/task/task.py]: done in different way

Comment (Anurag810) [reply erpnext/projects/doctype/task/task.py]: Now fixing travis

**Pull request closed without merging**